### PR TITLE
🐛 storage better logging

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

Storage was keeping a db connection while it's not required and it was timing out. This is no longer the case

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]
:+1:

File: services/storage/src/simcore_service_storage/simcore_s3_dsm.py [hunk 1]
```diff
@@ -514,9 +514,10 @@ async def delete_project_simcore_s3(
                 await db_file_meta_data.delete_all_from_project(conn, project_id)
             else:
                 await db_file_meta_data.delete_all_from_node(conn, node_id)
-            await get_s3_client(self.app).delete_files_in_project_node(
-                self.simcore_bucket_name, project_id, node_id
-            )
+
+        await get_s3_client(self.app).delete_files_in_project_node(
+            self.simcore_bucket_name, project_id, node_id
+        )
```
Comment (sanderegg) [inline services/storage/src/simcore_service_storage/simcore_s3_dsm.py:520]: hmm, ok. now comes the question.
What if the S3 client raises for whatever reason?? then the database lose synchronisation right?

**Review by sanderegg — CHANGES_REQUESTED** [review]
let's discuss the implications there

File: services/storage/src/simcore_service_storage/simcore_s3_dsm.py (hunk 1 above)
Comment (GitHK) [reply services/storage/src/simcore_service_storage/simcore_s3_dsm.py]: yes this will not bring anything. reverted and added more logging for now.

**Review by pcrespov — APPROVED** [review]
Besides the log, there is no change?

**Review by sanderegg — APPROVED** [review]

**Pull request merged**